systemtest/src/test/java/io/strimzi/systemtest/kafka/KafkaST.java

```diff
@@ -1342,6 +901,21 @@ void testMessagesAreStoredInDisk(ExtensionContext extensionContext) {
         );
         ClientUtils.waitForClientsSuccess(testStorage);
 
+        LOGGER.info("Verify presence of files created to store offsets topic");
+        String commandToGetFiles =  "cd /var/lib/kafka/data/kafka-log0/;" +
```

Comment:
How about `wc -l` to get count of files instead of looping and creating a String?